### DISCUSSION TITLE
Worked perfectly for over a year, now it's flatlined

### DISCUSSION THREAD

**Discussion by PerpSearch** [mrlt8/docker-wyze-bridge#1068] [Q&A] [answered]
I did the mrlt8 install a long time ago and it worked like a charm on 10 cams and the doorbell and it all worked on Synology Surveillance Stations as well.  Everything was going smoothly until I decided to change a cam name. I edited the stack in portainer as I had before and it crashed the container and there seems to be no getting it back.  I'm on a Synology DS723+ with the following stack and error msg.

version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554 # RTSP
            - 8888:8888 # HLS
            - 6050:6050 # WEB-UI
        environment:
            - DEBUG_LEVEL = debug
            - WYZE_EMAIL= XXXXXXXXXXXXXXXXX # Replace with wyze email 
            - WYZE_PASSWORD= XXXXXXXXXXXXX # Replace with wyze password 
            - FILTER_NAMES=Zoom, Northwest, North, Northeast, Southeast, Southwest

failed to deploy a stack: Network wyze_default Creating Network wyze_default Created Container wyze-bridge Creating Container wyze-bridge Created Container wyze-bridge Starting Error response from daemon: driver failed programming external connectivity on endpoint wyze-bridge (4f7ac217a2b96dd0edccf8d920a98d562b351b058ed98e9d21b9b7f5390d36b2): Error starting userland proxy: listen tcp4 0.0.0.0:1935: bind: address already in use . . . 

and  it goes on from there.

I also tried the version here on GetHub, but that one threw in a few more error because of addresses already in use 

version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554 # RTSP
            - 8888:8888 # HLS
            - 8889:8889 #WebRTC
            - 8189:8189/udp # WebRTC/ICE
            # - 5000:5000 # WEB-UI  (already in use) 
        environment:
            # [OPTIONAL] (Can be set in the WebUI):
            - WYZE_EMAIL=
            - WYZE_PASSWORD= 
            # [OPTIONAL] IP Address of the host to enable WebRTC e.g.,:
            # - WB_IP=192.168.1.122
            # [OPTIONAL] Wyze API credentials:
            # - API_ID=
            # - API_KEY= 

I'd appreciate any thoughts, but I'm a novice. Thanks.

**Comment by mrlt8** [comment, marked as answer]
Is the previous bridge still up or is something else already using those ports? 

you can specify a different port by changing the value before the `:` like this:
```yaml
ports:
- 1936:1935 # RTMP  (changed to 1936)
- 8554:8554 # RTSP
- 8888:8888 # HLS
- 8889:8889 #WebRTC
- 8189:8189/udp # WebRTC/ICE
- 5050:5000 # WEB-UI (changed to 5050)
```

**Reply by PerpSearch** [reply]
I wiped it out and tried again. Tried previous versions and nothing. Thank you so much for responding. I'll be trying this 1st thing in the AM and report back. I may not know much about this stuff but I can go where I'm pointed. Thanks again and chat later. 

Truly a Christmas miracle!  I may never understand, but happy just the same.  Thanks so much.
